testing/snapshots/rosetta.md

```diff
@@ -0,0 +1,951 @@
+|410   |a6bca1a0eb622eb91c89f4d5cf7c9a1b715d82a873849cfdeb9d630b8318d69b|2021-05-18T21:42:13.2430864Z|
+|411   |b3ab4ee99fbcf07e22deb36df345be5a586a33be5a6e21c0b0cbbf4147922bd7|2021-05-18T21:43:43.2430864Z|
+|412   |4fa6e4275c49e775444be79cd3720992356f41b142ddd1af18627a0fae094cd0|2021-05-18T21:45:13.2430864Z|
+|413   |79e6ae7f9d4af5748b8bb30d9ce5a9e9b6d2eb9a06d342769651a3f0a038bf82|2021-05-18T21:46:43.2430864Z|
+|414   |6af98fd5189c085b73959736f968a7906dbcbd854177cba25e474d1154a1c5e1|2021-05-18T21:48:13.2430864Z|
+|415   |369918440a47d15bfb19183110850763f5522e00f43ed08bd2fe327b58c07a5a|2021-05-18T21:49:43.2430864Z|
+|416   |52e2ded88f0fdf30397db54809db950f1706975e2402cf6cf29748343e270726|2021-05-18T21:51:13.2430864Z|
+|417   |e32535a90239da6e67708d669c934e3526f89159fdfd1eba5f8f9b143fb4d516|2021-05-18T21:52:43.2430864Z|
+|418   |94febd8543d078b26e0cf44c803304882df242325ab56bbf0fe3825096eea4ee|2021-05-18T21:54:13.2430864Z|
+|419   |f76ad4cbb3aee2b04791d47bc67f9d5b8d5aff230981ef96f6781083918d0fa2|2021-05-18T21:55:43.2430864Z|
+|420   |f4b4376b73219ff1af5b80731293877e78286fd673f878000cbe9d7a162a2896|2021-05-18T21:57:13.2430864Z|
+|421   |b43dc6b1373888e0c4eb2db0b6e9aae2de9ff2d3f142f29c179cd4aae2a8cc87|2021-05-18T21:58:43.2430864Z|
+|422   |8a0d2c88d0f6c8651d74116333bb97a25cb4c6fa10da663746f31b05f7870be7|2021-05-18T22:00:13.2430864Z|
+|423   |e4b2040342291c145c7d635946f44069d5469636cff75388b0a75e62b9a0a616|2021-05-18T22:01:43.2430864Z|
+|424   |6af26621eca92babda2df3ebcd2fe269946b3bf208183569258630e64486831d|2021-05-18T22:03:13.2430864Z|
+|425   |594d59b2e61bb18b149ffaac2b27b0efe1854f6795cd3bb96a443c3676d78683|2021-05-18T22:04:43.2430864Z|
+
+</details>
+
+## Events
+
+Event listing contains the data about the events in the index.
+For each block, one or more transactions are listed.
+Transactions are listed by their index.
+For each transaction, we list the events recorded during that transaction - the ID, event type, and any relevant data (e.g. in the case of transfer the account and the amount).
+Events are also ordered by their index within the transaction.
+
+<details>
+<summary> Show full event listing </summary>
+
+```
+- height: 13, block ID: af528bb047d6cd1400a326bb127d689607a096f5ccd81d8903dfebbac26afb23
+	0. transaction ID: a9c9ab28ea76b7dbfd1f2666f74348e4188d67cf68248df6634cee3f06adf7b1
+		0. AccountCreated: 0x754aed9de6197641                                                 (id: 84db285c44d1422fb75fec6ee522097c1498f86070eb9c977c40b56703344cd1)
+		1. flow.AccountKeyAdded                                                               (id: f29833fc5ff8a11a9d08c3cb81b20d4e8c3f825a5fec28bc85c30ac17d4a6750)
+		2. FlowToken.TokensWithdrawn: account=0x8c5303eaa26202d6, amount=100.00000000         (id: ba24b1dc1d90f0dcd614c75f58cd37c9bc69ef3fe7ed605624067b72b3eb45cd)
+		3. FlowToken.TokensDeposited: account=0x754aed9de6197641, amount=100.00000000         (id: 8e201802501b38118f2d082557fdf321c377ac618b29f51b0721632bacfe2b33)
```

Comment:
Once again I think that YAML format would suit this much nicer and make it more readable:

```yaml
blocks:
  -
    blockID: af528bb047d6cd1400a326bb127d689607a096f5ccd81d8903dfebbac26afb23
    height: 13
    transactions:
      -
        id: a9c9ab28ea76b7dbfd1f2666f74348e4188d67cf68248df6634cee3f06adf7b1
        events:
          -
            id: 84db285c44d1422fb75fec6ee522097c1498f86070eb9c977c40b56703344cd1
            type: AccountCreated
            address: 0x754aed9de6197641
          -
            id: f29833fc5ff8a11a9d08c3cb81b20d4e8c3f825a5fec28bc85c30ac17d4a6750
            type: flow.AccountKeyAdded
          -
            id: ba24b1dc1d90f0dcd614c75f58cd37c9bc69ef3fe7ed605624067b72b3eb45cd
            type: FlowToken.TokensWithdrawn
            account: 0x8c5303eaa26202d6
            amount: 100.00000000
          -
            id: 8e201802501b38118f2d082557fdf321c377ac618b29f51b0721632bacfe2b33
            type: FlowToken.TokensDeposited
            account: 0x754aed9de6197641
            amount: 100.00000000
```

On top of readability, it would make it easy to automatically generate the docs, which is always nice as no one likes maintaining old documentation :p